src/snyk/common/watchers/configurationWatcher.ts

```diff
@@ -51,9 +52,13 @@ class ConfigurationWatcher implements IWatcher {
       await extension.contextService.setContext(SNYK_CONTEXT.LOGGEDIN, false);
       await extension.contextService.setContext(SNYK_CONTEXT.AUTHENTICATION_METHOD_CHANGED, true);
       return extension.viewManagerService.refreshAllViews();
-    } else if (key === ADVANCED_CUSTOM_LS_PATH) {
+    } else if (key === ADVANCED_CLI_PATH) {
       // Language Server client must sync config changes before we can restart
       return _.debounce(() => extension.restartLanguageServer(), DEFAULT_LS_DEBOUNCE_INTERVAL)();
+    } else if(key === ADVANCED_CLI_RELEASE_CHANNEL) {
+      await extension.stopLanguageServer();
```

Comment:
remove stopLanguageserver from here after it's implemented in the cli downloader